<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagePowerWgtClass</class>
 <widget class="QWidget" name="PagePowerWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Current in hold mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="holdCurrentSpin">
     <property name="suffix">
      <string> %</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Current reduction delay</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="powerOffChk">
     <property name="text">
      <string>Power off when stop</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Power off delay</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="powerOffDelaySpin">
     <property name="suffix">
      <string> s</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="currentReductChk">
     <property name="text">
      <string>Current reduction enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="smoothCurrentChk">
     <property name="text">
      <string>Jerk free</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Current set time</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="currReductDelaySpin">
     <property name="suffix">
      <string> ms</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="smoothCurrentSpin">
     <property name="suffix">
      <string> ms</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>currentReductChk</tabstop>
  <tabstop>holdCurrentSpin</tabstop>
  <tabstop>currReductDelaySpin</tabstop>
  <tabstop>powerOffChk</tabstop>
  <tabstop>powerOffDelaySpin</tabstop>
  <tabstop>smoothCurrentChk</tabstop>
  <tabstop>smoothCurrentSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>powerOffChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>powerOffChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>powerOffDelaySpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentReductChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentReductChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentReductChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>holdCurrentSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentReductChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>currReductDelaySpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothCurrentChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>smoothCurrentSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothCurrentChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
